src/pudl/analysis/timeseries_cleaning.py

```diff
@@ -533,17 +533,7 @@ def impute_latc_tubal(  # noqa: C901
 
 
 class Timeseries:
-    """Multivariate timeseries for anomalies detection and imputation.
-
-    Attributes:
-        xi: Reference to the original values (can be null).
-            Many methods assume that these represent chronological, regular timeseries.
-        x: Copy of :attr:`xi` with any flagged values replaced with null.
-        flags: Flag label for each value, or null if not flagged.
-        flagged: Running list of flags that have been checked so far.
-        index: Row index.
-        columns: Column names.
-    """
+    """Multivariate timeseries for anomaly detection and imputation."""
```

Comment:
Having the attributes documented both in the class docstring and the body of the class definition was causing `sphinx-autoapi` 3.2 to put two copies in the docs, which created RST errors. See https://github.com/readthedocs/sphinx-autoapi/issues/476